gc_log.txt

```diff
@@ -0,0 +1,655 @@
+performing daemon worker op: 20
```

Comment:
Is this file used anywhere?

Reply:
Nope, just slipped through because `jujutsu` auto-`add`s every file that's not `.gitignore`d